packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5TradeLink.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from 'react';
+import { useCtraderServiceToken } from '@deriv/api';
+import { Button, Text } from '@deriv/quill-design';
+import { getPlatformFromUrl } from '../../../../../helpers/urls';
+import { THooks, TPlatforms } from '../../../../../types';
+import { AppToContentMapper, PlatformDetails, PlatformToLabelIconMapper, PlatformUrls } from '../../../constants';
+
+type TMT5TradeLinkProps = {
+    app?: keyof typeof AppToContentMapper;
+    isDemo?: THooks.ActiveAccount['is_virtual'];
+    platform?: TPlatforms.All;
+    webtraderUrl?: THooks.MT5AccountsList['webtrader_url'];
+};
+
+const MT5TradeLink: FC<TMT5TradeLinkProps> = ({ app = 'linux', isDemo = false, platform, webtraderUrl = '' }) => {
+    const content = AppToContentMapper[app];
+    const { data: ctraderToken } = useCtraderServiceToken();
+
+    const mt5Platform = PlatformDetails.mt5.platform;
+    const dxtradePlatform = PlatformDetails.dxtrade.platform;
+    const ctraderPlatform = PlatformDetails.ctrader.platform;
+
+    const onClickWebTerminal = () => {
+        const { isStaging, isTestLink } = getPlatformFromUrl();
+        let url;
+        switch (platform) {
+            case dxtradePlatform:
+                url = isDemo ? PlatformUrls.dxtrade.demo : PlatformUrls.dxtrade.live;
+                break;
+            case ctraderPlatform:
+                url = isTestLink || isStaging ? PlatformUrls.ctrader.staging : PlatformUrls.ctrader.live;
+                if (ctraderToken) url += `?token=${ctraderToken}`;
+                break;
+            default:
+                url = '';
+        }
+        window.open(url);
+    };
+
+    return (
+        <div className='flex justify-between items-center border-t border-[#f2f3f4] pt-4 px-6'>
```

Comment:
we have this f2f3f4 color in tailwind.config.js
Can use that

Reply:
You can use
border-system-light-secondary-background
```suggestion
        <div className='flex justify-between items-center border-t border-system-light-secondary-background pt-4 px-6'>
```